main.js

```diff
@@ -0,0 +1,23 @@
+let newTaskQuestion = prompt("タスクを入力してください");
+const taskLists = [];
+
+
+function makeNewTaskList(taskname, taskGenre) {
+  let newTaskName = taskname;
+  let newTaskGenre = taskGenre;
+  let newTask = {
+    "name": newTaskName,
+    "genre": newTaskGenre
+  };
```

Comment:
オブジェクトのkeyは文字列のようにせず、以下のようにするのが一般的です。
```suggestion
  let newTask = {
    name: newTaskName,
    genre: newTaskGenre
  };
```